### PR TITLE
Out with the stub in with the main

### PR DESCRIPTION
**Pull request by jarolrod**
We've advanced V1 enough; this does away with the stub file and renames it main.qml

[Windows](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/win64/insecure_win_gui.zip?branch=pull/159)
[Intel macOS](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos/insecure_mac_gui.zip?branch=pull/159)
[Apple Silicon macOS](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos_arm64/insecure_mac_arm64_gui.zip?branch=pull/159)
[ARM64 Android](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/android/insecure_android_apk.zip?branch=pull/159)

### CODE REVIEW

File: src/qml/pages/main.qml [hunk 1]
```diff
@@ -61,6 +61,6 @@ ApplicationWindow {
                     progress: nodeModel.verificationProgress
                 }
             }
-         }
+        }
```
Comment (hebasto) [inline src/qml/pages/main.qml:64]: To minimize chances of conflicts during repo syncing, suggesting to drop this change or move it into a separated commit.

**Review by hebasto — COMMENTED** [review]
Concept ACK.

File: src/qml/pages/main.qml (hunk 1 above)
Comment (jarolrod) [reply src/qml/pages/main.qml]: dropped

**Comment by jarolrod** [comment]
updated from b3b3966 to 447fd0e

changes: dropped formatting change to avoid future conflicts, rebased over main

**Review by hebasto — APPROVED** [review]
ACK 447fd0e459fe750f422b03138f24cd4de8a0b61f, I have reviewed the code and it looks OK, I agree it can be merged.

**Pull request merged**